Primo.js

```diff
@@ -312,15 +318,24 @@ function importPNX(text, url) {
 	fetchCreators(item, creators, 'author', splitGuidance);
 	fetchCreators(item, contributors, 'contributor', splitGuidance);
 	
-	var publisher = ZU.xpathText(doc, '//display/publisher');
-	if(publisher) var pubplace = ZU.unescapeHTML(publisher).split(" : ");
-	if(pubplace && pubplace[1]) {
-		item.place = pubplace[0].replace(/,\s*c?\d+(\-\d+)?|[\(\)\[\]]|(\.\s*)?/g, "");
-		item.publisher = pubplace[1].replace(/,\s*c?\d+(\-\d+)?|[\(\)\[\]]|(\.\s*)?/g, "")
-			.replace(/^\s*"|,?"\s*$/g, '');
-	} else if(pubplace) {
-		item.publisher = pubplace[0].replace(/,\s*c?\d+(\-\d+)?|[\(\[].+[\)\]]|(\.\s*)?/g, "")
-			.replace(/^\s*"|,?"?\s*$/g, '');
+	item.place = ZU.xpathText(doc, '//addata/cop');
+	var publisher = ZU.xpathText(doc, '//addata/pub');
+	if(!publisher) publisher = ZU.xpathText(doc, '//display/publisher');
+	if(publisher) {
+    	publisher = publisher.replace(/,\s*c?\d+|[\(\)\[\]]|(\.\s*)?/g, "");
+	    item.publisher = publisher.replace(/^\s*"|,?"\s*$/g, '');
+	    var pubplace = ZU.unescapeHTML(publisher).split(" : ");
+
+	    if(pubplace && pubplace[1]) {
+	        var possibleplace = pubplace[0];
+	        if(!item.place ) {
+	            item.publisher = pubplace[1].replace(/^\s*"|,?"\s*$/g, '');
+	            item.place = possibleplace;
+	        }
+	        if(item.place && item.place == possibleplace) {
+	            item.publisher = pubplace[1].replace(/^\s*"|,?"\s*$/g, '');
+	        }
+	    }
```

Comment:
I forgot why we have here to replace the `"` with some possible white spaces in the beginning and end. Do we have examples for that?